test/interactions/keyInteractionTests.ts

```diff
@@ -196,6 +196,12 @@ describe("Interactions", () => {
         TestMethods.triggerFakeKeyboardEvent("keyup", component.background(), aCode);
         assert.isFalse(aCallbackCalled, "callback for \"a\" was not called when \"a\" key was released");
 
+        TestMethods.triggerFakeMouseEvent("mouseout", component.background(), -100, -100);
+        TestMethods.triggerFakeKeyboardEvent("keydown", component.background(), aCode);
+        TestMethods.triggerFakeMouseEvent("mouseover", component.background(), 100, 100);
+        TestMethods.triggerFakeKeyboardEvent("keyup", component.background(), aCode);
+        TestMethods.triggerFakeKeyboardEvent("keydown", component.background(), aCode, { repeat : true });
+        assert.isFalse(aCallbackCalled, "callback for \"a\" was not called when \"a\" key was released");
```

Comment:
It's not clear how this differs from the above sequence of fake events and `assert()`. I would pull this out into a separate test for the exact functionality that was implemented, which is that `KeyPress` callbacks are only triggered once, even if there are multiple keydown events. Pseudocode:
```
Mouse over
Key Down
Assert callback called
Key Down again
Assert callback not called a second time
```